xapian-letor/api/letor_internal.h

```diff
@@ -23,38 +24,38 @@
 #define XAPIAN_INCLUDED_LETOR_INTERNAL_H
 
 #include "xapian-letor/letor.h"
-#include "xapian-letor/ranker.h"
-
-#include <map>
```

Comment:
`std::map` is used in this header, so it's better to `#include <map>` explicitly rather than relying on it happening to get included by another header included from here.  That way each header is more self-contained, and eliminating an apparently unnecessary include from one header doesn't trigger an avalanche of required changes to other files.